### PR TITLE
aliens aint shit

### PR DESCRIPTION
**Pull request by Kapu1178**
## About The Pull Request
aliens aint shit
https://www.youtube.com/watch?v=Gbccypt7twI
## Why It's Good For The Game
aliens aint shit
## Changelog
:cl:
soundadd: aliens aint shit
/:cl:

### CODE REVIEW

**Comment by chizzy376** [comment]
doesn't fit the filk theme other lobby songs have

**Comment by Kapu1178** [comment]
> doesn't fit the filk theme other lobby songs have

its not a lobby song you dolt

File: code/controllers/subsystem/ticker.dm [hunk 1]
```diff
@@ -127,6 +127,10 @@ SUBSYSTEM_DEF(ticker)
 				for(var/client/C AS in GLOB.clients)
 					C.mob?.update_sight() // To reveal ghosts
 
+				if(mode.round_finished == MODE_INFESTATION_M_MAJOR && prob(1))
```
Comment (ike709) [inline code/controllers/subsystem/ticker.dm:130]:
```suggestion
				if(mode.round_finished == MODE_INFESTATION_M_MAJOR && prob(5))
```

Comment (Kapu1178) [reply code/controllers/subsystem/ticker.dm:130]: mods can we get a vibe check

**Comment by chizzy376** [comment]
> > doesn't fit the filk theme other lobby songs have
> 
> its not a lobby song you dolt

sorry :(

**Comment by Lumipharon** [comment]
This is a meme pr, with a meme reason. We have actual round end music, this would literally be playing on top of it.

**Comment by TiviPlus** [comment]
see above

**Pull request closed without merging**